backend/prisma/migrations/20220930110053_temp_activity_log_fix/migration.sql

```diff
@@ -0,0 +1,10 @@
+-- AlterTable
+ALTER TABLE "ActivityLog" ADD COLUMN     "entityId" TEXT,
+ADD COLUMN     "userId" TEXT;
+
+
+UPDATE "ActivityLog" SET "entityId" = 'f3666617-01ec-4f7f-8bd9-6d337268c30a', "userId" ='2cf38670-0a8a-41e9-9018-e8b8a9b36486';
```

Comment:
Im not sure if this is really a "fix" for handling the addition of a new required field on `ActivityLog`

Reply:
hmm, it's not too ideal since this would update all activity logs to have this data. ideally, we would create a query that would set the entityId based on the entity name of the activity log but im not sure how we would handle the userId. this would probably be a pretty complex query so let's just leave this for now XD